iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/hint/DocumentOrganizationBuilderTest.java

```diff
@@ -0,0 +1,184 @@
+package eu.dnetlib.iis.wf.affmatching.hint;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.verify;
+
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.function.Function;
+import org.apache.spark.api.java.function.PairFunction;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import eu.dnetlib.iis.wf.affmatching.model.DocumentOrganization;
+import eu.dnetlib.iis.wf.affmatching.model.DocumentProject;
+import eu.dnetlib.iis.wf.affmatching.model.ProjectOrganization;
+import scala.Tuple2;
+
+/**
+ * @author mhorst
+ *
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class DocumentOrganizationBuilderTest {
+
+    DocumentOrganizationBuilder builder = new DocumentOrganizationBuilder();
+
+    @Mock
+    private JavaRDD<DocumentProject> documentProjectRdd;
+
+    @Captor
+    private ArgumentCaptor<PairFunction<DocumentProject, String, DocumentProject>> pairDocumentProjectFunctionArg;
```

Comment:
Here you finish the name of the function with Arg and in other classes you don't.